ll.tm

```diff
@@ -4877,6 +4882,11 @@ FuncAttr -> FuncAttr
 	| 'uwtable'
 	| 'willreturn'
 	| 'writeonly'
+	| 'hot'
+	| 'mustprogress'
+	| 'nocallback'
+	| 'noprofile'
+	| 'byref'
```

Comment:
Had a second look.

`byref` is not a function attribute, it is a parameter-only attribute (ParamAttr).

```cpp
    case lltok::kw_byref:
      HaveError |=
          error(Lex.getLoc(),
                "invalid use of parameter-only attribute on a function");
      break;
```